testing/asserts.ts

```diff
@@ -710,43 +731,59 @@ export function assertThrows<E extends Error = Error>(
  * callback which will be passed the error, usually to apply some custom
  * assertions on it.
  */
+export function assertRejects(
+  fn: () => Promise<unknown>,
+  msg?: string,
+): Promise<unknown>;
 export function assertRejects<E extends Error = Error>(
   fn: () => Promise<unknown>,
   // deno-lint-ignore no-explicit-any
-  ErrorClass?: new (...args: any[]) => E,
+  ErrorClass: new (...args: any[]) => E,
   msgIncludes?: string,
   msg?: string,
-): Promise<void>;
+): Promise<E>;
+export function assertRejects(
+  fn: () => Promise<unknown>,
+  msgIncludes: string,
+  msg?: string,
+): Promise<Error>;
```

Comment:
This signature seems having the same problem as the case of `assertThrows` (`msgIncludes` is not compared with the throws error message).

```ts
import { assertRejects } from "./testing/asserts.ts";
await assertRejects(
  async () => {
    throw new Error("foo");
  },
  "bar",
  "baz",
);
```

Reply:
Right, I see. I'll remove it as well. What do you think?

Reply:
Removing sounds good to me